core/standard/clause_11_Security_Considerations.adoc

```diff
@@ -0,0 +1,59 @@
+== Section 11: Security Considerations
+
+[width="90%",cols="2,6a"]
+|===
+|*Requirement {counter:req-id}* |/req/oas30/security +
```

Comment:
As this is a requirement in the OAS 3 conformance class, this would need to go to clause 9.

Reply:
After looking at this again, I think we need/should do the following:

* Move the OpenAPI specific parts to Clause 9 as part of the OpenAPI requirements class.
* Still add a Clause 11 "Security Considerations" and discuss general Web API vulnerabilities etc. Maybe there is existing text about this somewhere that we can reference. Highlight the importance of HTTPS/TLS. And reference the section in Clause 9.